packages/cli/src/command.js

```diff
@@ -99,6 +103,18 @@ export default class NuxtCommand {
     return new Generator(nuxt, builder)
   }
 
+  setLock(lockRelease) {
+    if (lockRelease) {
```

Comment:
Can we ensure former lock to be released here? (And possibly throwing an error)

Reply:
Dont but releasing the lock is async so now setLock is also async